release-notes/5.0/preview/api-diff/preview2/.Net/3.0-preview2_System.Text.Json.md

```diff
@@ -0,0 +1,237 @@
+# System.Text.Json
```

Comment:
/cc @layomia 

Maybe we should delete everything here related to writable DOM since it has been removed in master